build/checkstyle/alluxio_checks.xml

```diff
@@ -222,6 +222,7 @@
     <module name="JavadocMethod">
       <property name="id" value="ProductionScope"/>
       <property name="scope" value="public"/>
+      <property name="allowMissingThrowsTags" value="true"/>
```

Comment:
Is it possible to restrict the missing tags to only certain types of exceptions?